orcid-core/src/main/java/org/orcid/core/oauth/service/OrcidRandomValueTokenServicesImpl.java

```diff
@@ -280,7 +280,7 @@ public boolean longLifeTokenExist(String clientId, String userId, Collection<Str
         }
         
         for(OAuth2AccessToken token : existingTokens) {
-            if(!token.isExpired() && token.getExpiration().after(java.sql.Timestamp.valueOf(LocalDateTime.now().plusHours(1)))) {
+            if (token.getAdditionalInformation().get("persistent") != null && Boolean.valueOf((token.getAdditionalInformation().get("persistent").toString()))){
```

Comment:
Hey @TomDemeranville, could you please use the constant OrcidOauth2Constants.PERSISTENT instead of "persistent"? So, it will be easier to maintain. 

Thanks